logformat/rust/Cargo.toml

```diff
@@ -1,21 +1,23 @@
 [package]
 name = "logformat"
-version = "0.1.0"
+version = "0.2.0"
+edition = "2018"
+description = "definitions for data types used to serialize raw traces"
 
 [dependencies]
 rmp = "^0.8"
 enum-primitive-derive = "^0.1"
-num-traits = "^0.1"
-abomonation = "0.4"
+num-traits = "^0.2"
+abomonation = "0.7"
+abomonation_derive = "0.3"
+# @TODO: replace with serde
```

Comment:
We'll need support for `Abomonation` as long as we pass the types through timely.

Reply:
I thought we might be able to replace the `json` dependency with `serde_json` at some point, since from looking at the github it didn't seem like the `json` crate is still being actively maintained and we depend on serde at dev time anyways. That's not high prio for me though.

Could `abomonation` in theory be replaced with `Serialize / Deserialize` from `serde` when using `timely` with the `bincode` feature flag? I'm not sure how big the performance hit would be, but I know we're doing something like this in 3DF. Again, not high prio, just something I noted while updating the dependencies.